styles/globals.scss

```diff
@@ -16,8 +16,18 @@ body {
     max-width: 186px;
 }
 
-.top-actions-bar {
-    gap: 1rem;
+.column-gap-1 {
+    column-gap: 1rem;
+}
+
+.nfts-grid {
+    display: grid;
+    place-items: center;
```

Comment:
imo, it would look better if we removed the `place-items: center`

![image](https://user-images.githubusercontent.com/72078237/180190288-6edc3bc5-5761-4a1b-abaa-2d46f819d3fa.png)

Reply:
@evavirseda done!